test/rules/prefer_entry_point_imports/prefer_entry_point_imports_test.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
+
+@TestOn('vm')
```

Comment:
@incendial annotation must be on first line

Reply:
Fixed